.vscode/launch.json

```
@@ -10,9 +10,9 @@
             "request": "launch",
             "preLaunchTask": "build",
             // If you have changed target frameworks, make sure to update the program path.
-            "program": "${workspaceFolder}/src/content/Kentico.Kontent.Boilerplate/bin/Debug/net5.0/Kentico.Kontent.Boilerplate.dll",
+            "program": "${workspaceFolder}/src/content/Kontent.Ai.Boilerplate/bin/Debug/net5.0/Kontent.Ai.Boilerplate.dll",
```

update directory path

```suggestion
            "program": "${workspaceFolder}/src/content/Kontent.Ai.Boilerplate/bin/Debug/net6.0/Kontent.Ai.Boilerplate.dll",
```